modules/combined/test/tests/additive_manufacturing/check_element_addition_2D.i

```diff
@@ -0,0 +1,96 @@
+[Problem]
```

Comment:
It is not required, but it would be nice to run hit format on your input file. Install `moose-hit-format` on your Atom and use `command + shirt + h`.

Reply:
oh good to know. Thanks!